src/tests/ftest/deployment/basic_checkout.py

```diff
@@ -28,21 +27,49 @@ def test_basic_checkout(self):
         :avocado: tags=installation,basiccheckout
         """
         # local param
-        ec_ior_flags = self.params.get("ec_ior_flags", "/run/ior/*")
+        flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
+        apis = self.params.get("ior_api", '/run/ior/iorflags/*')
+        dfuse_mount_dir = self.params.get("mount_dir", "/run/dfuse/*")
+        transfer_block_size = self.params.get("transfer_block_size",
+                                              '/run/ior/iorflags/*')
+        obj_class = self.params.get("obj_class", '/run/ior/iorflags/*')
+        ec_obj_class = self.params.get("ec_oclass", '/run/ior/*')
+        mdtest_params = self.params.get("mdtest_params", "/run/mdtest/*")
 
-        # run tests
-        self.test_ior_small()
-        self.test_mdtest_small()
 
-        self.ior_cmd.flags.update(ec_ior_flags)
-        self.test_ec()
+        #run ior
+        results = self.run_ior_multiple_variants(obj_class, apis, transfer_block_size,
+                                                 flags, dfuse_mount_dir)
 
-        self.container.destroy()
-        self.pool.destroy()
+        #run ior with different ec oclass
+        results_ec = self.run_ior_multiple_variants(ec_obj_class, [apis[0]],
+                                                    [transfer_block_size[1]],
+                                                    [flags[0]], dfuse_mount_dir)
+        results = results + results_ec
+        self.log.error("Summary of IOR small test results:")
```

Comment:
```suggestion
        self.log.info("Summary of IOR small test results:")
```
Isn't this supposed to be info log?

Reply:
+1

Reply:
Thanks will do.

Reply:
Done.